Algorithm/QCAlgorithm.Trading.cs

```diff
@@ -46,19 +46,7 @@ public OrderTicket Buy(Symbol symbol, int quantity)
         /// Buy Stock (Alias of Order)
         /// </summary>
         /// <param name="symbol">string Symbol of the asset to trade</param>
-        /// <param name="quantity">double Quantity of the asset to trade</param>
-        /// <seealso cref="Buy(Symbol, int)"/>
-        public OrderTicket Buy(Symbol symbol, double quantity)
```

Comment:
The problem with having both double and decimal overloads is you get:

> Error	231	The call is ambiguous between the following methods or properties

Removing the double overload results in an implicit conversion to decimal and no error.